tools/js-sdk-release-tools/src/llc/apiVersion/apiVersionTypeExtractor.ts

```diff
@@ -14,12 +55,18 @@ export const getApiVersionType: IApiVersionTypeExtractor = async (
         const typeFromClient = await getApiVersionTypeFromRestClient(packageRoot, pattern, tryFindRestClientPath);
         if (typeFromClient !== ApiVersionType.None) return typeFromClient;
     }
-    
-    const parametersPath = join(packageRoot, "src/parameters.ts");
-    if (!(await exists(parametersPath))) {
-        throw new Error(`Failed to find parameters file '${parametersPath}'.`);
+
+    logger.info('No client found, fallback to get api version type in operation\'s parameter');
+    const parametersPath = await resolveParameterPath(packageRoot);
+    if (await exists(parametersPath)) {
+        const typeFromOperations = getApiVersionTypeFromOperations(parametersPath);
```

Comment:
i am not sure why we still need this code? get service api-version from parameters.ts file?

Reply:
yes. you can check out https://github.com/Azure/azure-sdk-tools/blob/main/tools/js-sdk-release-tools/src/common/changlog/automaticGenerateChangeLogAndBumpVersion.ts#L32